frontend_app_file/src/component/PreviewComponent.jsx

```diff
@@ -12,7 +14,7 @@ export class PreviewComponent extends React.Component {
     super(props)
     this.state = {
       displayLightbox: false,
-      isJpegPreviewDisplayable: true
+      jpegPreviewDisplayable: IMG_LOAD_STATE.LOADING
```

Comment:
renaming the variable as you change its type is good.
But you can go deeper in the renaming.
Here, it still implies a boolean, you can name it "jpegPreviewState" or "jpegPreviewLoadingState"